README.markdown

```diff
@@ -36,6 +36,7 @@
     - [Constants](#constants)
     - [Static Methods and Variable Type Properties](#static-methods-and-variable-type-properties)
     - [Optionals](#optionals)
+      - [Unused Optional Binding](#unused-optional-binding)
```

Comment:
perhaps we should move this to the `Control Flow` section instead? It seems more fitting because this applies to `guard`s and `if`s, right?

Reply:
yes it does!